bin/api.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -ex
 
+~/bin/check-hostname.sh
```

Comment:
I snuck in this `~/bin/check-hostname.sh` as well.  This is very useful when you clone a VM.  You don't want the clone to boot up and start trying to connect to the cluster using the keys from the original instance.  This hostname check blocks that from occuring